pkg/v3/config/config.go

```diff
@@ -63,6 +63,20 @@ type OffchainConfig struct {
 
 	// MaxUpkeepBatchSize is the max upkeep batch size of the OCR2 report.
 	MaxUpkeepBatchSize int `json:"maxUpkeepBatchSize"`
+
+	// LogProviderConfig holds configuration for the log provider
+	LogProviderConfig LogProviderConfig `json:"logProviderConfig"`
+}
+
+type LogProviderConfig struct {
+	// NumOfLogUpkeeps is the number of log upkeeps supported by the registry
+	NumOfLogUpkeeps uint32 `json:"numOfLogUpkeeps"`
+
+	// FastExecLogsHigh is the maximum number of logs that we are committed to processing for each upkeep
+	FastExecLogsHigh uint32 `json:"fastExecLogsHigh"`
+
+	// FastExecLogsLow is the minimum number of logs that we are committed to processing for each upkeep
+	FastExecLogsLow uint32 `json:"fastExecLogsLow"`
```

Comment:
I see this is not actually being passed to log provider. Is that expected?

Reply:
Yes, we aren't using this field yet but it was going to be added eventually so I added it now, since all three fields have been identified as part of the scaling design

Reply:
ok, would still be good to pass to factory.logProvider.SetConfig(commontypes.LogEventProviderConfig{
 		NumLogUpkeeps:    conf.LogProviderConfig.NumOfLogUpkeeps,
 		FastExecLogsHigh: conf.LogProviderConfig.FastExecLogsHigh,
 	}) below so that core has access to the config and can use it when it is ready

Reply:
Will be handled in following PRs